_posts/2017-06-07-hello-world.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Hello World!"
+date:   2017-06-07 20:32:13 +0000
+---
+
+Welcome to our brand new ACSS website!
+
+As well as the site, we've made a few changes to the way the society works and we're hoping they'll really improve your experience. Let's take a look...
+
+## Code of Conduct
+TODO
```

Comment:
```
We're introducing a Code of Conduct for all society-related interactions. All members are expected to follow it whenever they attend an event, or participate in any official communication channels (Slack, Facebook, etc). We recommend you take a minute to have a read through - it can be found [here][CoC].
```